WORKING_GROUPS.md

```diff
@@ -1,11 +1,11 @@
 # Node.js Working Groups
 
 Node.js Working Groups are autonomous projects created by the
-[Technical Steering Committee (TSC)](https://github.com/nodejs/node/blob/master/GOVERNANCE.md#technical-committee).
+[Core Technical Committee (CTC)](https://github.com/nodejs/node/blob/master/GOVERNANCE.md#core-technical-committee).
```

Comment:
There exist non-core Working Groups in the Node.js Foundation, which report to the TSC. There can also be other working groups that report to other Top Level Projects (e.g. Express or libuv).
I don't think it's strictly necessary for this doc to describe the whole hierarchy, and things that it doesn't have charter, though it could certainly be improved (perhaps with a link). Maybe room for another PR? :)